### PR TITLE
Add YKB_TRACER docs

### PR DESCRIPTION
**Pull request by Pavel-Durov**
Related to https://github.com/ykjit/yk/pull/939
Update build_config docs with YKB_TRACER info.

### CODE REVIEW

File: docs/src/dev/build_config.md [hunk 1]
```diff
@@ -28,6 +28,13 @@ within a repository using `yk` (including the `yk` repository itself). If you
 want to set/unset `YKB_YKLLVM_BIN_DIR` then `cargo clean` any repositories
 using `yk` before rebuilding them.
 
+## `YKB_TRACER`
+
+`YKB_TRACER` environment variable allows building yk with either `hwt` (Hardware Tracer) or `swt` (Software Software Tracer).
```
Comment (ltratt) [inline docs/src/dev/build_config.md:33]: Can we wrap lines to the same length as others in the file?

[Also probably start this sentence with "The".]

**Comment by ltratt** [comment]
Please squash.

**Comment by ltratt** [comment]
Please squash.

**Comment by Pavel-Durov** [comment]
squashed 👉 5b9245e0db76cb2586bb162a5bf4e3d938089598

**Comment by Pavel-Durov** [comment]
I think it should be removed from the queue until https://github.com/ykjit/yk/pull/939 is merged

**Comment by Pavel-Durov** [comment]
https://github.com/ykjit/yk/pull/939 was merged 🥳 
We can merge this one as well

**Pull request merged**